pkgs/by-name/lo/louvain-community/package.nix

```diff
@@ -1,22 +1,26 @@
 { stdenv
 , fetchFromGitHub
+, fetchpatch
```

Comment:
Can be removed now

Reply:
Oh, great catch! I keep forgetting about stale imports. Pushed the removal.